generators/ci-cd/templates/_Jenkinsfile

```diff
@@ -156,6 +156,27 @@ node {
 <%_ if (jenkinsIntegrations.includes('docker')) { _%>
 <%= gitLabIndent %>    }
 <%_ } _%>
+<%_ if (jenkinsIntegrations.includes('publishDocker')) { _%>
+
+<%= gitLabIndent %>    def dockerImage
+<%= gitLabIndent %>    stage('build docker') {
+    <%_ if (buildTool === 'maven') { _%>
+<%= gitLabIndent %>        sh "cp -R src/main/docker target/"
```

Comment:
That's mainly because when you build in docker you don't have docker in the build container. And I would want to avoid docker-in-docker and the complications that go with it.
Now we could use docker:build when not building in docker. That would just be a little more work.